data/all.json

```diff
@@ -1292,6 +1292,42 @@
 				"android"
 			],
 			"order": 0
+		},
+		{
+			"name": "Internet connection status bar",
+			"description": "Want to notify user if the device is offline? Display Internet connection status bar",
+			"screenshots": [
+				"https://raw.githubusercontent.com/NativeScript/code-samples/master/screens/internet-connection-status-bar.gif"
+			],
+			"links": {
+				"angular": "https://play.nativescript.org/?template=play-ng&id=1RycDo&v=10"
+			},
+			"categories": [
+				"layouts and pages"
+			],
+			"authors": [
+				{
+					"name": "Aleksander Rakutska",
+					"npmuser": "arakutska",
+					"framework": "angular"
+				},
+				{
+					"name": "Jakub Pawlak",
+					"npmuser": "pawlak",
+					"framework": "angular"
```

Comment:
Currently there is no way to show double authors for the same framework :( . You need to remove one of those as only the first one will be used when showed under the https://market.nativescript.org section.

The authors array is designed to include 3 elements, for each "framework"